testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/ForbiddenUrlTestCase.java

```diff
@@ -98,13 +100,32 @@ public void testAllowedUrl() throws Exception {
     public void testForbiddenUrl() throws Exception {
         HttpClientBuilder httpClientBuilder = HttpClientBuilder.create();
 
-        try(CloseableHttpClient client = httpClientBuilder.build()) {
+        try (CloseableHttpClient client = httpClientBuilder.build()) {
 
-            HttpUriRequest getVarRequest = new HttpGet(url.toExternalForm() + FORBIDDEN_URL);
+            HttpUriRequest getVarRequest = new HttpGet(url.toExternalForm() + jakartafiUrl(FORBIDDEN_URL));
             try (CloseableHttpResponse getVarResponse = client.execute(getVarRequest)) {
                 int statusCode = getVarResponse.getStatusLine().getStatusCode();
-                Assert.assertEquals("Status code should be " + FORBIDDEN_STATUS_CODE + ", but is " + statusCode + ". For more information see JBEAP-18842. ", FORBIDDEN_STATUS_CODE, statusCode);
+                Assert.assertEquals("Status code should be " + FORBIDDEN_STATUS_CODE + ", but is " + statusCode +
+                        ". For more information see JBEAP-18842. ", FORBIDDEN_STATUS_CODE, statusCode);
             }
         }
     }
+
```

Comment:
@jamezp @bstansberry, This hack is a bit ugly, but, other than continuing to ignore the test, I'm not sure how best to go about it. That said, the test does now pass in main and WFP as expected.  Also, IDEA cleaned up some formatting and import order that I didn't notice before pushing. I don't mind the clean up, but if you'd like less noise, I can reset and make a quieter change for this. :)

Reply:
I assume `javax.faces.resource` has special meeting in which case I can't think of a better way to handle it.

Reply:
It is. It's a mean to access bundled resources in META-INF, iirc.